geemusic/intents/selection.py

```diff
@@ -94,6 +95,51 @@ def play_artist_radio(artist_name):
     speech_text = "Playing %s radio" % artist['name']
     return audio(speech_text).play(stream_url)
 
+@ask.intent("GeeMusicPlayPlaylistIntent")
+def play_playlist(playlist_name):
```

Comment:
I might refactor this to be a bit more functional to match the rest of the code (basically just replace any iteration with `map` and `filter`), but this looks good to me otherwise 👍